Catalogs/Vehicles/VehicleCatalog.xosc

```diff
@@ -3,76 +3,192 @@
   <FileHeader revMajor="1" revMinor="0" date="2020-07-10T10:00:00" author="BMW AG" description="Vehicle catalog" />
   <Catalog name="VehicleCatalog">
     <Vehicle name="car_ego" vehicleCategory="car">
+	  <Properties>
+        <Property name="AirDragCoefficient" value="0.3"/>
+        <Property name="AxleRatio" value="1.0"/>
+        <Property name="DecelerationFromPowertrainDrag" value="0.5"/>
+        <Property name="FrictionCoefficient" value="1.0"/>
+        <Property name="FrontSurface" value="2.4"/>
+        <Property name="GearRatio1" value="9.6"/>
+        <Property name="Mass" value="1300.0"/>
+        <Property name="MaximumEngineSpeed" value="6000.0"/>
+        <Property name="MaximumEngineTorque" value="250.0"/>
+        <Property name="MinimumEngineSpeed" value="900.0"/>
+        <Property name="MinimumEngineTorque" value="-54.0"/>
+        <Property name="MomentInertiaPitch" value="0.0"/>
+        <Property name="MomentInertiaRoll" value="0.0"/>
+        <Property name="MomentInertiaYaw" value="0.0"/>
+        <Property name="NumberOfGears" value="1"/>
+        <Property name="SteeringRatio" value="10.0"/>
+      </Properties>
       <BoundingBox>
         <Center x="1.4" y="0.0" z="0.9" />
         <Dimensions width="2.0" length="5.0" height="1.8" />
       </BoundingBox>
       <Performance maxSpeed="70" maxDeceleration="10" maxAcceleration="10" />
       <Axles>
-        <FrontAxle maxSteering="0.5" wheelDiameter="0.8" trackWidth="1.68" positionX="2.98" positionZ="0.4" />
+        <FrontAxle maxSteering="30" wheelDiameter="0.8" trackWidth="1.68" positionX="2.98" positionZ="0.4" />
```

Comment:
This is 0.5 because the OSC standard says the angle is in rad and not in degree.

Reply:
If this is necessary for openPASS to run correctly we can change it to 30 and make a comment that this is necessary for openPASS in degree but according to the standard actually should be rad. esmini is not affected by he change. We can change it back to 0.5 once openPass can handle it.

Reply:
Added comment in the VehicleCatalog.xosc in 536ead5b8057f58b086da519c8d476cde1f3ff4c